packages/react-dom/src/__tests__/ReactServerRenderingHydration.js

```diff
@@ -258,6 +258,45 @@ describe('ReactDOMServerHydration', () => {
     expect(element.firstChild.focus).not.toHaveBeenCalled();
   });
 
+  it('should warn when the style property differs', () => {
+    const element = document.createElement('div');
+    element.innerHTML = ReactDOMServer.renderToString(
+      <div style={{textDecoration: 'none', color: 'black'}} />,
+    );
+    expect(element.firstChild.style.textDecoration).toBe('none');
+    expect(element.firstChild.style.color).toBe('black');
+
+    expect(() =>
+      ReactDOM.hydrate(
+        <div style={{textDecoration: 'none', color: 'white'}} />,
+        element,
+      ),
+    ).toWarnDev(
+      'Warning: Prop `style` did not match. Server: ' +
+        '"text-decoration:none;color:black" Client: ' +
+        '"text-decoration:none;color:white"',
+      {withoutStack: true},
+    );
+  });
+
+  it('should not warn when the style property differs on whitespace only', () => {
```

Comment:
I'm not sure how useful this test is; it'd need to be run in IE to make sure there is no regression, other browsers (and jsdom) would pass it even without this PR's changes.

Maybe it's useful as a way of ensuring my changes don't break jsdom?

Reply:
Is there anything in jsdom you can monkeypatch for this specific test to reproduce IE behavior? I agree it's kind of useless like this.